internal/auth/metrics.go

```diff
@@ -40,7 +40,8 @@ func GetActionTag(req *http.Request) string {
 	}
 	// get the action from the url path
 	path := req.URL.Path
-	if action, ok := pathToAction[path]; ok {
+	pathBase := strings.Split(path, "/")
+	if action, ok := pathToAction["/"+pathBase[len(pathBase)-1]]; ok {
```

Comment:
can we do this on the previous line and pull out some of this logic. I also _strongly_ prefer `fmt.Sprintf` accurately

```go
splitPath := strings.Split(req.URL.PAth, "/")
path := fmt.Sprintf("/%s", splitPath[len(splitPath) - 1])
if action, ok := pathToAction[path]; ok {
. . .
}

Reply:
That’s fair! 👍